code/common/finspace.q

```diff
@@ -54,9 +54,9 @@ notifyhdb:{[cluster;changeset]
       
       // Ensuring that the changeset has successfully created before doing the HDB reload
       current:.finspace.checkstatus[(`.aws.get_changeset;.finspace.database;changeset[`id]);("COMPLETED";"FAILED");00:01;0wu];
-      .lg.o[`notifyhdb;("notifying ",string[cluster]," to repoint to changeset ",changeset[`id])];
-      awsdb:.aws.db[.finspace.database;changeset[`id];.finspace.cache;.finspace.dataview];
-      .aws.update_kx_cluster_databases[string[cluster];.aws.sdbs[awsdb];.aws.sdep[.finspace.hdbreloadmode]]
+      //.lg.o[`notifyhdb;("notifying ",string[cluster]," to repoint to changeset ",changeset[`id])];
```

Comment:
Can we just delete this line if it is no longer needed? We can always recreate it later if we need it